MAUI/Autocomplete/Getting-Started.md

```diff
@@ -174,36 +199,45 @@ Now, populate this 'SocialMediaViewModel' data in the [Autocomplete](https://hel
 {% highlight C# %}
 
 SocialMediaViewModel socialMediaViewModel = new SocialMediaViewModel();
-autocomplete.BindingContext = socialMediaViewModel;
-autocomplete.ItemsSource = SocialMediaViewModel.SocialMedias;
+this.BindingContext = socialMediaViewModel;
+SfAutocomplete autocomplete = new SfAutocomplete(); 
+autocomplete.WidthRequest = 250;
+autocomplete.HeightRequest = 50;
+autocomplete.ItemsSource = socialMediaViewModel.SocialMedias;
+Content = autocomplete;
 
 {% endhighlight %}
 {% endtabs %}
 
-N> Set the 'SocialMediaViewModel' instance as the `BindingContext` of your control. This is done to bind the properties of 'SocialMediaViewModel' to the `Autocomplete`.
+N> Set the 'SocialMediaViewModel' instance as the **BindingContext** of your control. This is done to bind the properties of 'SocialMediaViewModel' to the Autocomplete.
```

Comment:
Set the BindingContext of your page to an instance of SocialMediaViewModel. This allows you to bind the properties of SocialMediaViewModel to the Autocomplete control.

Reply:
Updated.